filebeat/filebeat_parsers_test.go

```diff
@@ -0,0 +1,132 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//go:build linux && cgo && withjournald
```

Comment:
@kvch do I need to add the `integration` flag here?

Reply:
We do not need such a high-level test for journald input in beats. I would rather we put such tests in `e2e-testing` repository. For testing parsers in journald input, you should check out how it is done for the filestream input: https://github.com/elastic/beats/blob/master/filebeat/input/filestream/parsers_integration_test.go

Regardless, I would get this PR in without this test, and add integration tests in a follow up PR. WDYT?